FWCore/Framework/interface/ESConsumesCollector.h

```diff
@@ -38,8 +39,10 @@
 #include <vector>
 #include <memory>
 namespace edm {
-  using ESConsumesInfo =
-      std::vector<std::tuple<edm::eventsetup::EventSetupRecordKey, edm::eventsetup::DataKey, std::string> >;
+  using ESConsumesInfo = std::vector<std::tuple<edm::eventsetup::EventSetupRecordKey,
```

Comment:
It could. Given it is an internal detail never seen by users and highly restricted in its use internally, I do not feel any strong need to use a struct. The type was defined before this pull request and only slightly modified.

@makortel what are your thoughts?

Reply:
I think it could make the code a bit easier to follow, since there are `std::get<>()` calls in other files.

Reply:
By quick look I agree with @kpedro88 that the code using `ESConsumesInfo` would be easier to follow if it would contain a struct instead of a tuple. If the `std::get<>()` side would use either types or named constants I'd be less sure on changing to struct.